src/renderer/browser/util/getRandomIV.js

```diff
@@ -0,0 +1,5 @@
+import { randomBytes } from 'crypto';
+
+export default function getRandomIV() {
+  return randomBytes(16);
```

Comment:
Would we maybe want the ability to choose the strength?
Having an optional parameter with 128 as default, and having` return randomBytes(strength /8);`